src/portable/espressif/esp32s2/dcd_esp32s2.c

```diff
@@ -71,6 +71,20 @@ static uint32_t _setup_packet[2];
 #define XFER_CTL_BASE(_ep, _dir) &xfer_status[_ep][_dir]
 static xfer_ctl_t xfer_status[EP_MAX][2];
 
+// Max number of IN EP FIFOs
+#define USB_FIFO_NUM 5
```

Comment:
To be consistent, this should be name `EP_FIFO_NUM` and should be placed next to `EP_FIFO_SIZE`